src/components/PetForm.js

```diff
@@ -15,10 +15,11 @@ export default class PetForm extends Component {
   constructor(props) {
     super(props);
     this.onFileChange = this.onFileChange.bind(this);
+    this.imageStatusChange = this.imageStatusChange.bind(this);
     this.state = {
       breeds: [],
       name: '',
-      typeOfAnimal: 'Cat',
+      typeOfAnimal: 'Select',
```

Comment:
Changed the default to "select" but it's not a big deal.

Reply:
I checked to make sure that a user wasn't able to submit the form with "select" selected in the dropdown. Since there are no breeds for "select", the form is unable to be submitted. So I think this works!

Reply:
I would change the default to '' as well as the option value (The text can still be 'Select'), cause otherwise if it's selected and the user tries to submit, the browser won't remind the user that this field is not selected. You can do so with the Sex as well.

Reply:
Fixed the default to be ''.
If the `Type Of Animal` is `Select`, `Breed` is always blank and won't let the user to submit. But I fixed the default to avoid the confusion!